codice/ddf
# DDF-05019 Visual design updates for search sidebar.

What are the relevant tickets?
Fixes: #5019

#### What does this PR do?
The main purpose is to make visual updates to the search sidebar. 
- align search selector and results
- move 'source health' icon to the row of icons
- split icons and overflow menu in search selector
- removes background color on the search selector dropdown
- frames filter/sort area and left align it
- brings search results forward on darker themes
- adds border to right side of sidebar

#### Who is reviewing it? 
@adimka @djblue @vinamartin @maryformanek @lifenouveau @michaelmenousek 

#### Ask 2 committers to review/merge the PR and tag them here.
@adimka @djblue @vinamartin


#### How should this be tested?
Boot DDF, go to Workspaces, Run a search. Verify that changes only affected the search sidebar and that they work on a variety of theme colors. 

#### Screenshots
![image](https://user-images.githubusercontent.com/11765980/60552011-14fa4a00-9ce2-11e9-819d-fa75daf7a29d.png)

![image](https://user-images.githubusercontent.com/11765980/60552032-23e0fc80-9ce2-11e9-924a-76048c1b7a30.png)


#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
